GetProfile/handler.ts

```diff
@@ -44,7 +44,8 @@ type IGetProfileHandler = (
  */
 export function GetProfileHandler(
   profileModel: ProfileModel,
-  optOutEmailSwitchDate: Date
+  optOutEmailSwitchDate: Date,
+  isOptOutEmailEnabled: boolean
```

Comment:
rename this property because we want the new logic when we activate _opt-in_ on services.
```suggestion
  isOptInEmailEnabled: boolean
```